props/nested-components.md

```diff
@@ -0,0 +1,195 @@
+# Nested Components
+
+## Goals
+
+In order to increase the complexity of our React webapps, we should practice nesting components.
+
+Nested components share a relationship that is different from sibling components.
+
+We can apply a design pattern for components that will help us distinguish what our components do.
+
+## Format
+
+This lesson will take the following steps:
+
+1. Introduce two design patterns for components:
+   - presentational components
+   - container components
+1. Follow Sofia's journey on creating an attendance app, applying these design patterns
```

Comment:
this construction parallels the common phrasing of "journey to ..." like journey to the west, or journey to the stars, so it sounds more expected to me.
```suggestion
1. Follow Sofia's journey to create an attendance app, applying these design patterns
```